Dockerfile

```diff
@@ -3,12 +3,21 @@ FROM python:3.7-alpine3.9
 ENV PYTHON_UNBUFFERED 1
 ENV PYTHONWARNINGS "ignore:Unverified HTTPS request"
 
-RUN apk add --update git ca-certificates && rm -rf /var/cache/apk/*
+# Create user and homedir
+RUN set -x \
+    && addgroup -g 101 -S indexer \
+    && adduser -S -D -u 101 -h /home/indexer -s /sbin/nologin -G indexer -g indexer indexer
+
+RUN apk add --no-cache --update git ca-certificates
 
 ADD requirements.txt /
+RUN pip install --upgrade pip
 RUN pip install -r /requirements.txt
 
 WORKDIR /app
 COPY indexer.py /app/
 COPY mapping.json /app/
+
+USER indexer
```

Comment:
```suggestion
USER 101
```

Reply:
Why? It works both ways

Reply:
A "security" feature (a nominal user could still have uid 0 and be root)

https://kubernetes.io/docs/concepts/policy/pod-security-policy/#users-and-groups